### ISSUE TITLE
Questions about the correspondence between the published pre-trained models and the algorithms in the paper.

### ISSUE THREAD

**Issue by muchen12** [hitachinsk/FGT#32]
I am sorry to bother you.
There are some questions about the correspondences between the published pre-trained models and the algorithms in the paper. I noticed that your published pre-trained model list have 'lafc' and 'lafc_single' two flow completion weights. At the same time, as shown in paper named 'Exploiting Optical Flow Guidance for Transformer-Based Video Inpainting', Tab. 4 provided 'S' 'LA' 'LA+Le' three algorithms to compare with each other. I am sorry to bother you to consult about which two of the published two weights correspond to two of the three comparison algorithms respectively.
Thanks very much. Looking forward to your reply soon!

**Comment by hitachinsk** [comment]
“S” represents the "lafc_single" model, and "LA+le" is the "lafc" model. "LA" is the model used for ablation study to demonstrate the effectiveness of edge loss, I did not share this model because I don't think this is the final checkpoint.

**Comment by muchen12** [comment]
Got it. Thank you very much.

**Comment by hitachinsk** [comment]
You got it.
